src/locales/default.json

```diff
@@ -989,6 +989,11 @@
         "withdraw": {
             "priceImpact": "Price impact from removing liquidity results when the value of each token removed is not proportional to the weights of the pool. Removing non-proportional amounts causes the internal prices of the pool to change, as if you were swapping tokens. The higher the price impact, the worse price you’ll get for exiting your position.",
             "total": "The total value in {0} you’ll be withdrawing from this pool."
+        },
+        "veBalApr": {
+            "title": "Locking / veBAL APR",
```

Comment:
@garethfuller @pkattera  We might want to word this as "Max locking/veBAL APR" or "Max veBAL APR" for short

Reply:
Happy with either of those suggestions.

Reply:
Updated